config/urls.py

```diff
@@ -84,6 +84,7 @@
             kwargs={"exception": Exception("Page not Found")},
         ),
         path("500/", default_views.server_error),
+        path("api-auth/", include("rest_framework.urls")),
```

Comment:
This seems like a change that is not associated with this PR, can you undo this change?

Reply:
For debugging, use the basic authentication method.